packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx

```diff
@@ -5,7 +5,7 @@ import { useAppointmentsMetrics } from './appointment-metrics.resource';
 import MetricsCard from './metrics-card.component';
 import MetricsHeader from './metrics-header.component';
 import styles from './appointments-metrics.scss';
-import { ErrorState } from '@openmrs/esm-framework';
+// import { ErrorState } from '@openmrs/esm-framework';
```

Comment:
Good work.
I suggest that you drop the commented out line

Reply:
Done